src/basic/NESTED_INIT.cpp

```diff
@@ -26,21 +26,21 @@ NESTED_INIT::NESTED_INIT(const RunParams& params)
 {
   m_n_init = 100;
 
-  setDefaultSize(m_n_init * m_n_init * m_n_init);
+  setDefaultProblemSize(m_n_init * m_n_init * m_n_init);
   setDefaultReps(1000);
 
-  auto n_final = std::cbrt(getRunSize());
+  auto n_final = std::cbrt(getTargetProblemSize())+1;
```

Comment:
Why +1?

Reply:
I saw that +1 was added in a bunch of other places, I guess to protect against size = 0 for small problem runs. I'm not sure why it is needed in some places but not others. I will remove this one if you don't think it is needed.

Reply:
I don't think its needed here because cbrt(n) doesn't get to zero until n < 1.